xfuser/model_executor/models/transformers/transformer_flux.py

```diff
@@ -109,21 +109,20 @@ def forward(
             else self.time_text_embed(timestep, guidance, pooled_projections)
         )
         encoder_hidden_states = self.context_embedder(encoder_hidden_states)
-
-        if txt_ids.ndim == 3:
-            logger.warning(
-                "Passing `txt_ids` 3d torch.Tensor is deprecated."
-                "Please remove the batch dimension and pass it as a 2d torch Tensor"
-            )
-            txt_ids = txt_ids[0]
-        if img_ids.ndim == 3:
-            logger.warning(
-                "Passing `img_ids` 3d torch.Tensor is deprecated."
-                "Please remove the batch dimension and pass it as a 2d torch Tensor"
-            )
-            img_ids = img_ids[0]
-
-        ids = torch.cat((txt_ids, img_ids), dim=0)
+        # if txt_ids.ndim == 3:
```

Comment:
why delete the check

Reply:
In diffuser v0.30.3, it supports passing three dimensions tensor. So we do not need to lower the dimension of the tensor.
https://github.com/huggingface/diffusers/blob/v0.30.3/src/diffusers/models/transformers/transformer_flux.py#L380